app/models/match.rb

```diff
@@ -38,119 +38,118 @@ class Match < ActiveRecord::Base
+        lambda { |team|
+          includes(contester1: :team, contester2: :team)
+            .where("teams.id = ? OR teams_contesters.id = ?", team.id, team.id)
+        }
   scope :of_userteam,
-    lambda { |user, team| {
-    :include => {:matchers => {:contester => :team}},
-    :conditions => ["teams.id = ? AND matchers.user_id = ?", team.id, user.id]} }
+        lambda { |user, team|
+          includes(matchers: { contester: :team })
+            .where("teams.id = ? AND matchers.user_id = ?", team.id, user.id)
+        }
   scope :within_time,
-    lambda { |from, to| {
-    :conditions => ["match_time > ? AND match_time < ?", from.utc, to.utc]} }
+        ->(from, to) { where("match_time > ? AND match_time < ?", from.utc, to.utc) }
   scope :around,
-    lambda { |time| {
-    :conditions => ["match_time > ? AND match_time < ?",  time.ago(MATCH_LENGTH).utc, time.ago(-MATCH_LENGTH).utc]} }
+        lambda { |time|
+          where("match_time > ? AND match_time < ?",
+                time.ago(MATCH_LENGTH).utc, time.ago(-MATCH_LENGTH).utc)
+        }
   scope :after,
-    lambda { |time| {
-    :conditions => ["match_time > ? AND match_time < ?",  time.utc, time.ago(-MATCH_LENGTH).utc]} }
+        ->time { where("match_time > ? AND match_time < ?", time.utc, time.ago(-MATCH_LENGTH).utc) }
   scope :map_stats,
-    :select => "map1_id, COUNT(*) as num, maps.name",
-    :joins => "LEFT JOIN maps ON maps.id = map1_id",
-    :group => "map1_id",
-    :having => "map1_id is not null",
-    :order => "num DESC"
+        select: "map1_id, COUNT(*) as num, maps.name",
+        joins: "LEFT JOIN maps ON maps.id = map1_id",
+        group: "map1_id",
+        having: "map1_id is not null",
+        order: "num DESC"
   scope :year_stats,
-    :select => "id, DATE_FORMAT(match_time, '%Y') as year, COUNT(*) as num",
-    :conditions => "match_time > '2000-01-01 01:01:01'",
-    :group => "year",
-    :order => "num DESC"
+        select: "id, DATE_FORMAT(match_time, '%Y') as year, COUNT(*) as num",
+        conditions: "match_time > '2000-01-01 01:01:01'",
+        group: "year",
+        order: "num DESC"
   scope :month_stats,
-    :select => "id, DATE_FORMAT(match_time, '%m') as month_n,
-    DATE_FORMAT(match_time, '%M') as month,
-    COUNT(*) as num",
-    :conditions => "match_time > '2000-01-01 01:01:01'",
-    :group => "month",
-    :order => "month_n"
+        select: "id, DATE_FORMAT(match_time, '%m') as month_n,
+        DATE_FORMAT(match_time, '%M') as month,
+        COUNT(*) as num",
+        conditions: "match_time > '2000-01-01 01:01:01'",
+        group: "month",
+        order: "month_n"
 
   validates_presence_of :contester1, :contester2, :contest
-  validates_format_of [:score1, :score2], :with => /\A[0-9]\z/, :allow_nil => true
-  validates_length_of :report, :maximum => 64000, :allow_blank => true
+  validates_format_of [:score1, :score2], with: /\A[0-9]\z/, allow_nil: true
```

Comment:
Prefer the new style validations `validates :column, format: value` over `validates_format_of`.